iroh-metrics/src/portmap.rs

```diff
@@ -44,6 +50,12 @@ impl Default for Metrics {
             upnp_gateway_updated: Counter::new(
                 "Number of UPnP probes that resulted in a gateway different to the previous one.",
             ),
+
+            /*
+             * PCP metrics
+             */
+            pcp_probes: Counter::new("Number of PCP probes executed."),
+            pcp_available: Counter::new("Number of PCP probes that found it available."),
```

Comment:
who or what is `it` here?

Reply:
PCP? how would you phrase it?

Reply:
phrasing is fine, I was just having some trouble understanding "it" ;)